### PR TITLE
Feat/edit release form

### PR DESCRIPTION
**Pull request by kaliabadi**

### CODE REVIEW

**Comment by kaliabadi** [comment]
pls squash

File: src/bin/actions/updateReleaseAction.js [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
+export default async (userDetails, options ) => {
+  console.log(options.release);
+  
+  const repoDetails = getOrgRepo();
 
-export default async (userDetails, version) => {
-  const updateVersion = version ?
-    version :
-    await new Promise((resolve) => gitTags.get((err, tags) => resolve(tags[0])));
+  if (!(options.version instanceof String)) {
```
Comment (shendriksen) [inline src/bin/actions/updateReleaseAction.js]: Does this check work for you? I think I had to change it to 'options.constructor === String' on my branch as it was a primitive string.

File: src/bin/github/gitInteractions.js [hunk 2]
```diff
@@ -18,28 +19,33 @@ const latestRelease = async (userDetails) => {
...
+  const versionNumber = await new Promise((resolve) => {
     gitTags.get((err, tags) => {
-      if (err){
-        reject(err);
-      } else {
-        if(!tags || tags.length === 0) console.error('You have not tagged your commit with the release version!')
-        resolve(tags[0]);
-      }
+      if (err) 
+        throw err;
```
Comment (shendriksen) [inline src/bin/github/gitInteractions.js]: should this part have been changed to a 'throw' from the 'reject'?

File: src/bin/github/gitInteractions.js [hunk 3]
```diff
@@ -58,20 +64,34 @@ const newRelease = async (userDetails, {approved, scheduled}) => {
...
   if(!taggedRelease) 
     throw new Error('❌ No release found from that tag ❌');
 
-  if(changeLogContents) {
-    const releaseBody = generateBodyContent(scheduled, approved, changeLogContents);
-    Object.assign(releaseDetails, { body: releaseBody });
-  }
+  fs.writeFileSync('./tmp/taggedRelease.json', JSON.stringify(taggedRelease.body));
+
+  var openInEditor = require('open-in-editor');
```
Comment (shendriksen) [inline src/bin/github/gitInteractions.js]: Does this need to be required here?

**Pull request merged**